pkgs/development/python-modules/fvcore/default.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, iopath
+, numpy
+, pillow
+, pyyaml
+, shapely
+, tabulate
+, termcolor
+, torch
+, torchvision
+, tqdm
+, yacs
+}:
+
+buildPythonPackage rec {
+  pname = "fvcore";
+  disabled = pythonOlder "3.6";
+  version = "0.1.5";
+
+  src = fetchFromGitHub {
+    owner = "facebookresearch";
+    repo = "fvcore";
+    rev = "6bde555";
+    sha256 = "1vz9qif5iaxhwks5mhg9lrfr16aj5kb108dgphhcv0iq4nla2wsa";
+  };
+
+  nativeBuildInputs = [
+    iopath
+    numpy
+    pillow
+    pyyaml
+    tabulate
+    termcolor
+    tqdm
+    yacs
+  ];
+
+  propagatedBuildInputs = [
+    torch
```

Comment:
```suggestion
```

[They consider it a "dependency"](https://github.com/facebookresearch/fvcore/blob/0f2b23b6f93e36041d9a74764ee824541cf0a0e5/.circleci/config.yml#L44), but it is only needed [in `fcvore.nn` and in tests](https://github.com/search?q=repo%3Afacebookresearch%2Ffvcore+%2Fimport+torch%2F&type=code). 

Consider adding it as an optional dependency, as with shapely.